src/DataService/cookieService.js

```diff
@@ -1,22 +1,22 @@
 function saveCookie(key, value) {
-    document.cookie = `${key}=${value}`;
+  document.cookie = `${key}=${value}`;
 }
 
 function loadCookie(key) {
-    let cookie = document.cookie;
-    cookie = cookie.substring(cookie.indexOf('=') + 1);
-    return cookie;
+  let cookie = document.cookie;
+  cookie = cookie.substring(cookie.indexOf('=') + 1);
+  return cookie;
```

Comment:
a bit of improvement
```suggestion
  const cookie = document.cookie;
  return cookie.substring(cookie.indexOf('=') + 1);;
```

Reply:
done